todo/js/script.js

```diff
@@ -136,84 +271,33 @@ const app = new function IIFE() {
             span = task.el.querySelector(`.${presets.taskElementClass.text}`);
         }
 
-        // in case we just want to update only value or state not both at the same time
+        // in case we just want to update only value or isDone not both at the same time
         // since undefined is primitive type we don't have to worry about type coercion
         if (value != undefined) {
             task.value = value;
 
             span.childNodes[0].nodeValue = value;
         }
 
-        if (state != undefined) {
-            task.state = state;
+        if (isDone != undefined) {
```

Comment:
Very fine suggestion it can be used on `value` ( few line before ) since only `falsy value` here would be `empty string` or `undefined` ( which makes it no need of default value ) , but for `isDone` since this is called in `update function` and it can be `false` ( must work with `false` value ) or `undefined` ( also `true` ), what would you suggest with having in mind I created this conditions to avoid running unnecessary part of the `update function` so it can update only parts it needed, or should I adjust my method? ( `isDone != undefined` what really made me use this line was I needed to detect `undefined` as well as `falsy value`. )